### ISSUE TITLE
[Accessibility] [Out logged Home][Code] Incorrect HTML language

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#859]
**Describe the bug**
When access muda’s page, the html language should be in portuguese or change as the user changes the navigation language. It impacts the screen readers behaviour.

**To Reproduce**
1. Go to 'https://muda.cambiatus.io/join?redirect=%2Fdashboard'
2. Inspect HTML
3.  Check HTML language keeps 'en' even when page language changes

**Expected behavior**
If the language is portuguese, the html must be “pt-br” too. In this way, the screen reader verbalizes as native. The same behaviour must work for all languages

**Screenshots**

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser  Chrome
 - Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**
It probably reflects on smartphones.

**Comment by ceciliabelem** [comment]
For mapping all defect statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
